routers/repo/commit.go

```diff
@@ -246,6 +247,15 @@ func Diff(ctx *context.Context) {
 	ctx.Data["Parents"] = parents
 	ctx.Data["DiffNotAvailable"] = diff.NumFiles() == 0
 	ctx.Data["SourcePath"] = setting.AppSubURL + "/" + path.Join(userName, repoName, "src", "commit", commitID)
+
+	note := git.Note{}
```

Comment:
This would be more correct:
```suggestion
	note := &git.Note{}
```